### DISCUSSION TITLE
polyvid.io - synced narrations for YouTube videos in any language

### DISCUSSION THREAD

**Discussion by pikil** [quasarframework/quasar#9161] [Show and Tell]
Hi all,
Just showing another project made with Awesome Quasar! This time it's Quasar V2!

https://polyvid.io/

Briefly: The project allows you to enter the link to almost any YouTube video and get the synced audio translation in your language!

While still in Alpha, it allows you to get the general idea of what the modern development with Quasar can do within a short amount of time :)

Any feedback, critics, suggestions - happy to discuss!

**Comment by ontwikkelfabriek** [comment]
Very nice. I noticed some translated text that's not displaying correctly in your app, 

https://polyvid.io/video/yt/BBOnU52CuI4/en/male ( 31 sec mark):
_Many people, after this training period, join the world of work and others decide to continue a little more, one or two years, doing what is called, an official **master&#39;s** degree._ ( the apostrophe display's as `&#39;s`)

**Reply by pikil** [reply]
Thank you for that! I'll make sure, there are no such gremlins hanging around with the next update :) Cheers!
